src/site/_includes/layout.njk

```diff
@@ -19,30 +32,26 @@ tags: []
 
     {% Meta locale, page, collections, renderData %}
 
-    <link rel="preconnect" href="//www.gstatic.com" crossorigin="" />
-    <link rel="preconnect" href="//www.google-analytics.com" crossorigin="" />
-    <link rel="preconnect" href="//firebaseremoteconfig.googleapis.com" crossorigin="" />
-    <link rel="preconnect" href="//firebaseinstallations.googleapis.com" crossorigin="" />
-    <link rel="preconnect" href="//webdev.imgix.net" crossorigin="" />
-
-    <link rel="preload" as="style" href="{{ resourceCSS.path | stripQueryParamsDev }}" />
-    {# We only preload Material Icons because the site can't render properly without them. #}
-    {# For Google Sans we use font-display: swap because preloading too many fonts can hurt performance. #}
-    {# https://www.zachleat.com/web/preload/ #}
-    <link rel="preload" as="font" crossorigin href="/fonts/material-icons/regular.woff2">
-
-    <link rel="stylesheet" href="{{ resourceCSS.path | stripQueryParamsDev }}" />
     <link rel="manifest" href="/manifest.webmanifest" />
     {# Include default icon even though we have a manifest #}
     <link rel="shortcut icon" href="/images/favicon.ico">
     <link rel="icon" type="image/png" sizes="32x32" href="/images/favicon-32x32.png">
     <link rel="icon" type="image/png" sizes="16x16" href="/images/favicon-16x16.png">
     <link rel="apple-touch-icon" sizes="180x180" href="/images/apple-touch-icon.png">
     <link rel="mask-icon" color="#0054ff" href="/images/safari-pinned-tab.svg">
-    <script async src="//www.google-analytics.com/analytics.js"></script>
-    <script defer src="{{ resourceJS.path | stripQueryParamsDev }}"></script>
+    <script type="module" src="{{ helpers.hashForProd('/js/app.js') }}"></script>
+    {# Add a facility for pages to declare an array of script paths. #}
+    {# If no scripts are declared we will use the default page script. #}
+    {# default.js loads the basic set of custom elements that all pages need. #}
+    {% if pageScripts %}
+      {% for item in pageScripts %}
+        <script type="module" src="{{ helpers.hashForProd(item) }}"></script>
+      {% endfor %}
```

Comment:
I think nunjucks allows else from a for loop too (if nothing was output, do something else). You can streamline this.